Apps/W1/Shopify/app/src/Base/Pages/ShpfyShopCard.Page.al

```diff
@@ -974,6 +997,25 @@ page 30101 "Shpfy Shop Card"
                         Report.Run(Report::"Shpfy Sync Shipm. to Shopify");
                     end;
                 }
+                action(SyncPostedSalesInvoices)
+                {
+                    ApplicationArea = All;
+                    Caption = 'Sync Posted Sales Invoices';
+                    Image = Export;
+                    Promoted = true;
+                    PromotedCategory = Category5;
```

Comment:
Shouldn't new development uses the new syntax for promoted actions?

Reply:
I agree with you, bet then I would need to refactor all of the other actions on this page. So right now I won't do it here, but I will do it on the new pages that are in the PR.